### DISCUSSION TITLE
Split Screen while Full Screen

### DISCUSSION THREAD

**Discussion by chrislafay** [mrkai77/Loop#426] [Q&A]
I use multiple desktops frequently. On those multiple desktops, I use the "full screen/split screen" functionality.

I'd love the ability to be able to go full-screen with a couple of apps and move them around with keyboard commands.

One way that this could be accomplished using some core functionality already built: First, arrange your desktop with the windows how you want them. Once you have your windows they way you want them,  press a hotkey that transitions those windows to a "new desktop."

The two main issues I see with this methodology are (and I'm sure there are more):

1. There could be other windows on the screen at the time of "moving them to a new desktop"
2. If you "split the screen" with anything other than vertical dividers, those wouldn't transition over

I only wanted to post this methodology to help get the creative process going (as it's definitely not the final solution 😅).

---

👉🏻 Before I posted this discussion, I posted a comment on [this issue](https://github.com/MrKai77/Loop/issues/40#issuecomment-2179292661) to make sure it hasn't been duplicated anywhere yet.

**Comment by mrkai77** [comment]
Moved to #427.
